src/DataProtection/DataProtection/src/Microsoft.AspNetCore.DataProtection.csproj

```diff
@@ -14,6 +14,8 @@
 
   <ItemGroup>
     <Compile Include="..\..\shared\src\*.cs" />
+    <Compile Include="$(SharedSourceRoot)PlatformAttributes.cs"
+             Condition="'$(TargetFramework)' != '$(DefaultNetCoreTargetFramework)'" />
```

Comment:
nit: Plz avoid this kind of indentation in `msbuild` files. Just indent 2 or 4 spaces and be done w/ it.

Reply:
Could I address this feedback in a follow up (including updating the SDK)? I'd really like to get this PR in

Reply:
It was a nit❕😺 Don't worry about it.